src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -3355,7 +3378,16 @@ static void window_ride_construction_show_special_track_dropdown(rct_window* w,
         {
             auto ride = get_ride(_currentRideIndex);
             if (ride != nullptr && ride->type != RIDE_TYPE_SPINNING_WILD_MOUSE && ride->type != RIDE_TYPE_STEEL_WILD_MOUSE)
-                trackPieceStringId = STR_BOOSTER;
+            {
+                if (ride != nullptr
+                    && (ride->type == RIDE_TYPE_MINIATURE_RAILWAY || ride->type == RIDE_TYPE_CAR_RIDE
```

Comment:
Don't add massive if checks like this. We have the ride type descriptor and it's associated flags for this sort of thing.